lib/plugins/deploy/README.md

```diff
@@ -1,12 +1,14 @@
 # Deploy
 
 ```
-serverless deploy
+serverless deploy [function]
 ```
```

Comment:
Oh never mind. Now I see what's going on. It's the same doc for service deployment.